client/src/assets/leo1.svg

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Try minifying this svg.

Reply:
Thanks for the advice, I turned it into an issue (#14 ) 🔖